lib/ShareManifests.js

```diff
@@ -0,0 +1,108 @@
+    "name": "Facebook",
+    "origin": "https://www.facebook.com",
+    "pageSize": {
+      "share": {
+        "height": 400,
+        "width": 700
+      }
+    },
+    "shareURL": "https://www.facebook.com/sharer/sharer.php?u=%{url}",
+    "version": 2,
+    "postActivationURL": "https://activations.cdn.mozilla.net/en-US/activated/facebook.html"
+  },
+  "social.manifest.mg-mail.yahoo.com": {
+    "author": "Yahoo! Inc",
+    "description": "Quickly send web pages to yourself and others with enhanced link previews and more.",
+    "homepageURL": "http://mail.yahoo.com",
+    "icon32URL": "https://s.yimg.com/nq/nr/icons/ymail32.png",
+    "icon64URL": "https://s.yimg.com/nq/nr/icons/ymail64.png",
+    "iconURL": "https://s.yimg.com/nq/nr/icons/ymail16.png",
```

Comment:
We should not have any icon pulling from the web since they are directly used in fx ui.  I'll also look at why the build scripts for the activation site didn't catch those.